doc/litespeed-configuration.rst

```diff
@@ -0,0 +1,94 @@
+.. _litespeed configuration:
+
+LiteSpeed Configuration
+-----------------------
+
+Below you will find detailed LiteSpeed configuration recommendations for the
+features provided by this library.
+
+Preamble
+~~~~~~~~
+
+First of all, let's get one thing straight here: You'll find a lot of documentation
+and noise around LiteSpeed cache on the Internet, mostly involving plugins, specifically the
+Wordpress one. You **don't** need any plugin to benefit from LiteSpeed cache!
+As long as you follow the HTTP specification regarding the caching headers, you can use it as
+a general reverse proxy like NGINX or Varnish.
+
+Invalidation works by setting the specific LiteSpeed headers on the **response**. This means
+contrary to other proxies in this library, we do not send any ``PURGE`` requests to
+the proxy but instead we have to send a request to an endpoint where the response provides
+the correct LiteSpeed-specific headers which then trigger purging actions.
+You can read more on these headers in the `LiteSpeed response headers documentation`_.
+
+To do so, we generate a simple PHP file with a random file name containing the appropriate ``header()`` calls.
+After generation, we request this file and delete it again right away.
+
+For this reason you have to configure two parameters:
+
+* The location on your server where the file should be generated to (must be publicly accessible).
+* The base URL on which the generated file can be requested.
+
+Configuring LiteSpeed WebServer itself
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Enabling LiteSpeed to support public caching on your server is pretty much straight forward.
```

Comment:
how straight forward this is depends on how familiar you are with stuff ;-)

it seems this is an apache module? so if you are currently on nginx, its not straightforward at all... also, some people love configuring apache in .htaccess while others only do it in the /etc files.

pet peeve aside: should we explain how to set up and configure LiteSpeed in the FOSHttpCache documentation? i feel we should not, it seems out of scope to me. we should only talk about specifics that are relevant for us (like: what about multi server setup? would you then setup LightSpeed on an apache that acts as proxy? or have a http cache on each node? (in which case you'd need to send the request to each node and have the file on each node)

Reply:
okay, got it. then lets say we need to enable caching in the LiteSpeed server and explain only that part.

lets not talk about ESI here as FOSHttpCache is not about ESI. (if you want to add a note with just a link to their doc, or a detailed blog post about LiteSpeed ESI & Symfony, that would be fine)

Reply:
Right, will remove the part about ESI and Symfony.